slack-base/src/main/java/com/hubspot/slack/client/models/interaction/ViewSubmissionIF.java

```diff
@@ -0,0 +1,32 @@
+package com.hubspot.slack.client.models.interaction;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.SlackChannel;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ViewSubmissionIF extends SlackInteractiveCallback {
+
+  ViewInteractionPayload getView();
+
+  @Override
+  default SlackChannel getChannel() {
```

Comment:
We should come back and fix the class hierarchy at some point, this is definitely going to burn people =/